app/build.gradle

```diff
@@ -130,7 +130,6 @@ dependencies {
 
   implementation project(path: ':feature:audio')
 
-  implementation deps.kotlin.stdlib
```

Comment:
Yay, one less dependency! )

Reply:
still needed but it's transparently pulled in now by the Kotlin plugin